variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "member_group_name" {
+  description = "Name to give to the security group of kubernetes server"
+  type = string
+  default = ""
```

Comment:
I would not put `""` as a default value as this makes the variable optional and it can't be. We absolutely need an appropriate `value` for name to create the resource.